Sources/Kitura/RouterRequest.swift

```diff
@@ -32,7 +32,7 @@ public class RouterRequest: SocketReader {
     /// The hostname of the request
     ///
     public var hostname: String {
-        if  let host = headers["host"] {
+        if  let host = headers.get("host")?.first {
```

Comment:
Yeah it is more verbose but you are getting more from it. The new headers are case-insensitive and can handle multiple headers of the same name. I think in the long run this can help prevent hard to diagnose bugs.